installer/x86_64/install.sh

```diff
@@ -424,6 +429,7 @@ image_dir="image-$image_version"
 if [ "$install_env" = "onie" ]; then
     eval $create_demo_partition $blk_dev
     demo_dev=$(echo $blk_dev | sed -e 's/\(mmcblk[0-9]\)/\1p/')$demo_part
+    echo $blk_dev | grep -q nvme0 && demo_dev=$(echo $blk_dev | sed -e 's/\(nvme[0-9]n[0-9]\)/\1p/')$demo_part
```

Comment:
I do not really understand the code intention here. Please have a detailed description on what you are trying to achieve here.

Reply:
Hi Sir,
We will echo blk_dev value and check if nvme0 is existed. (echo $blk_dev | grep -q nvme0 &&)
If  nvme0 is existed, do to replace demo_dev to nvme.(demo_dev=$(echo $blk_dev | sed -e 's/\(nvme[0-9]n[0-9]\)/\1p/')$demo_part)
If  nvme0 is not existed, keep originally demo_dev value.(demo_dev=$(echo $blk_dev | sed -e 's/\(mmcblk[0-9]\)/\1p/')$demo_part)

Actually This logic come from ONIE: https://github.com/opencomputeproject/onie/commit/ed5108d023d23a876767f0311a8bc43d67d34bd6
![nvme](https://user-images.githubusercontent.com/19216597/68274088-1ded9480-00a3-11ea-80d6-f13b114b00d2.jpg)
If this is readable issue, we can modify like above.

Thanks a lot